uber/site_sections/registration.py

```diff
@@ -416,11 +416,12 @@ def register(self, session, message='', **params):
                     message = 'Please enter a real emergency contact number'
                 elif attendee.age_group == AGE_UNKNOWN:
                     message = 'Please select an age category'
```

Comment:
I realized this wasn't working while testing; it only required that something be present and didn't validate that it even looked like an email address.